src/tests/multihost/alltests/test_automount_from_bash.py

```diff
@@ -0,0 +1,860 @@
+        modify_gid = [(ldap.MOD_REPLACE, 'nisMapEntry', f'-fstype=nfs,rw {master_server}:/export/shared1/key_new'.encode('utf-8'))]
+        ldap_inst.modify_ldap(f'cn=key1,nisMapName=auto.share1,ou=mount,{ds_suffix}', modify_gid)
+        clear_only_domain_log(multihost)
+        client.run_command("sss_cache -a auto.share1 --domain=example1 --debug 10 2> /tmp/cache_debugout")
+        for result in ["failed", "No cache object matched the specified search", "No such autofs map"]:
+            with pytest.raises(AssertionError):
+                find_logs(multihost, "/tmp/cache_debugout", result)
+        client.run_command("ldbsearch -H /var/lib/sss/db/cache_example1.ldb name=auto.direct dataExpireTimestamp | grep dataExpireTimestamp: | grep -v -w 1")
+        client.run_command("ldbsearch -H /var/lib/sss/db/cache_example1.ldb name=auto.master dataExpireTimestamp | grep dataExpireTimestamp: | grep -v -w 1")
+        client.run_command("ldbsearch -H /var/lib/sss/db/cache_example1.ldb name=auto.home dataExpireTimestamp | grep dataExpireTimestamp: | grep -v -w 1")
+        client.run_command("ldbsearch -H /var/lib/sss/db/cache_example1.ldb name=auto.share1 dataExpireTimestamp | grep dataExpireTimestamp: | grep -w 1")
+        client.run_command("ldbsearch -H /var/lib/sss/db/cache_example1.ldb name=auto.share2 dataExpireTimestamp | grep dataExpireTimestamp: | grep -v -w 1")
+        client.run_command("ldbsearch -H /var/lib/sss/db/cache_example1.ldb name=auto.share3 dataExpireTimestamp | grep dataExpireTimestamp: | grep -v -w 1")
+        client.run_command("automount -m")
+        time.sleep(2)
+        find_logs(multihost, log_sssd, f"{master_server}:/export/shared1/key_new")
+
+    @staticmethod
+    def test_bz811987(multihost, backupsssdconf, ldap_autofs):
+        """
+        :title: maximum key name must be PATH MAX bz811987
+        :id: 4e9e6930-9897-11ed-b12f-845cf3eff344
+        :steps:
+            1. Create a big size user for automount
+            2. Mount it and should work as normal.
+        :expectedresults:
+            1. Should Success
+            2. Should Success
+        """
+        # maximum key name must be PATH MAX bz811987
+        client = multihost.client[0]
+        master_server = multihost.master[0].sys_hostname
+        ldap_uri = 'ldap://%s' % (multihost.master[0].sys_hostname)
+        ds_rootdn = 'cn=Directory Manager'
+        ds_rootpw = 'Secret123'
+        ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf("domain/example1", {'entry_cache_autofs_timeout': "60"}, action='update')
+        tools.clear_sssd_cache()
+        user_dn = f'cn=/largepathfolder1/largepathfolder2/largepathfolder3/largepathfolder4/largepathfolder5/largepathfolder6/largepathfolder7/largepathfolder8/largepathfolder9/largepathfolder10/largepathfolder11/largepathfolder12/largepathfolder13/largepathfolders,nisMapName=auto.direct,ou=mount,{ds_suffix}'
```

Comment:
These lines are too long even for my widescreen monitor. 
Please keep it at least under 119 charachters.